.github/workflows/close-old-issues.yaml

```diff
@@ -10,14 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       issues: write  # Ensure necessary permissions for issues
-      pull-requests: none
-      contents: none
     steps:
     - name: Close inactive issues
       uses: actions/stale@v9.0.0
       with:
+        # A list of labels to reference when looking through issues,
+        # and only when one (or even more) of these labels are found..
+        # then skip this issue, and never try to stale and/or close it.
         exempt-issue-labels: "Keep Issue Open"
-        days-before-issue-close: 14
-        close-issue-message: "This issue was closed because it has been inactive for 14 days"
-        debug-only: false # Make this field equal true if you want to test your configuration if it works or not
+        # Split it into two weeks, after one week the issue will be marked as stale,
+        # after another week have pasted without any update.. the issue will then be closed.
+        days-before-issue-stale: 7
+        days-before-issue-close: 7
```

Comment:
Had to split the time period of 2 weeks into:
- 1 week of inactivity -> mark issue as stale
- 1 week of inactivity (after marking the issue as stale) -> close issue

Reason behind this is because I don't know how to force this `actions/stale` GitHub Action to behave like the old JavaScript Code we had previously (close issue after 2 weeks of inactivity), if anyone knows a better way then this (while still using this GitHub Action).. then you go ahead and do so, test it, and make a PR for it 👍